docs/privacy-policy.md

```diff
@@ -0,0 +1,172 @@
+Privacy Policy
+==============
+
+**DRAFT**
+
+Your personal privacy is an important factor that the Sempervirens project and Numfocus 
+(our legal entity) considers in the development of each of our products and services. 
+We are committed to NOT storing ANY personal information through Sempervirens other 
+than IP addresses. We are committed to being transparent and open. This Privacy Policy 
+explains generically how we receive information from you, and what we do with that 
+information once we have it.
+
+What do we mean by "personal information?"
+------------------------------------------
+
+The phrase "personal information" means information which identifies you, such as your 
+name, email address, twitter account names, or other user names. Any information that 
+falls outside of this is "non-personal information."
+
+Combining non-personal information with personal information we consider to be 
+"personal information." Such combinations will not be publically shared. Removing all 
```

Comment:
I think that this document needs to begin with a set of defintions that includes things like "collect" and "release" etc.